packages/browser/src/core/inspector/index.ts

```diff
@@ -0,0 +1,15 @@
+import type { Inspector } from '@segment/inspector-core'
+
+declare global {
+  interface Window {
+    __SEGMENT_INSPECTOR__?: Inspector
+  }
+}
+
+// FIXME: Replace this with `globalThis` to support all environments
```

Comment:
Can we use this instead? https://github.com/segmentio/analytics-next/blob/master/packages/browser/src/lib/get-global.ts#L2

Reply:
cool. didn't know about this. fixed up!